public/scripts/app.js

```diff
@@ -63,6 +63,7 @@ function removeLocation(evt) {
     delete weatherApp.selectedLocations[parent.id];
     saveLocationList(weatherApp.selectedLocations);
   }
+  parent.remove();
```

Comment:
This is great - good catch, thank you!

Can you remove line 62 (`parent.setAttribute('hidden', true);`), and replace it with the `parent.remove()` instead?

Reply:
Of course :)

Reply:
Awesome, looks good, thanks!